tracer/src/Datadog.Trace/Debugger/Helpers/StringExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// <copyright file="StringExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Datadog.Trace.Debugger.Helpers
+{
+    internal static class StringExtensions
+    {
+        // https://stackoverflow.com/questions/18021808/uuid-interop-with-c-sharp-code
+        public static string ToUUID(this string input)
+        {
+            byte[] hash;
+            using (var md5 = MD5.Create())
+            {
+                hash = md5.ComputeHash(Encoding.UTF8.GetBytes(input));
+            }
```

Comment:
Related - for data streams monitoring we need to use the FNV hash. I wonder if we should use the same hash function here?

Reply:
If it's not a stopper, I wish we could leave it as is for this PR, since it was checked as working 😄 
The next PR I will work on - RCM compliance, where I rethink usage of MD5 hash, and consider to use your suggestion.